app/javascript/packs/containers/QuestionnaireRoutes.jsx

```diff
@@ -26,8 +29,8 @@ class QuestionnaireRoutes extends React.Component {
         <hr />
 
         <Switch>
-          <Route path={`${currentPath}/edit`}
-                 render={renderWithProps(AnswerModeContainer, propsToPass)} />
+          <Route path={`${currentPath}/edit/:cId?`}
+                 component={renderWithProps(AnswerModeContainer, propsToPass)} />
           <Route path={`${currentPath}/delegate`}
```

Comment:
Make sure this `component` prop naming change gets in for the rest of the routes here!